README.md

```diff
@@ -1 +1,98 @@
-# House-Rental-API
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# House Rental
```

Comment:
- [ ] You are using Rspec, but the "test" folder still exists, which is used by another test framework.  Kindly delete that folder if you are not using it. 👍